src/syscalls.perf.template

```diff
@@ -0,0 +1,82 @@
+%{
+/**
+ * Enhanced Seccomp x86_64 Syscall Table
+ *
+ * Copyright (c) 2012, 2020 Red Hat <pmoore@redhat.com>
+ * Author: Paul Moore <paul@paul-moore.com>
+ * gperf support: Giuseppe Scrivano <gscrivan@redhat.com>
```

Comment:
This copyright block isn't quite right, it makes it seem as though I updated it in 2020, which I did not.  I would expect something like the following (the authors part is flexible):

```
 * Copyright (c) 2012 Red Hat <pmoore@redhat.com>
 * Copyright (c) 2020 Red Hat <gscrivan@redhat.com>
 * Authors:  Paul Moore <paul@paul-moore.com>, Giuseppe Scrivano <gscrivan@redhat.com>
```